src/main/resources/feedbackQuestionRubricEditFormTableOptionsFragment.html

```diff
@@ -0,0 +1,16 @@
+<td class="align-center rubricCol-${questionIndex}-${col}" data-col="${col}">
+    <div class="btn-group">
+        <button type="button" class="btn btn-default" id="rubricMoveChoiceLink-${questionIndex}-${col}-l" onclick="swapRubricCol(${col}, ${questionIndex}, true)"
+            data-toggle="tooltip" data-placement="top" title="Move column left">
+            <span class="glyphicon glyphicon-arrow-left"></span>
+        </button>
+        <button type="button" class="btn btn-default" id="rubricRemoveChoiceLink-${questionIndex}-${col}" onclick="removeRubricCol(${col}, ${questionIndex})"
```

Comment:
@VamsiSangam yes, unfortunately older code does not seem to follow the style. An attempt was made to change the style of older code (see #5679 for an example) but was ultimately not done, and given the current level of activity from core members, GSoC students and additional contributors, it may be difficult to fix this right now without causing conflicts. Still, it would be better to follow the conventions now to reduce the amount of fixing that will need to be done later on.